### ISSUE TITLE
p.107, last line of the Chapter on Interpreters

### ISSUE THREAD

**Issue by alexhkurz** [andreasabel/plt-errata#10]
"An expression compiler c is correct if, for all expressions e, e ⇓ w if and only if c(e) ⇓ v."

Should the "w" be a "v"? What is the relationship between w and v?

**Comment by andreasabel** [comment]
> Should the "w" be a "v"?

Yes, but where is this quote from?  On p. 107 of the book, we are in the middle of Chapter 6 on code generation.

**Comment by alexhkurz** [comment]
I referred to the digital copy that is (or at least used to be) online.

I now checked with the hardcopy and the error does not appear there.

Thanks.

**Comment by andreasabel** [comment]
Ok, thanks anyway, Alexander!
